### PR TITLE
MRG DOC: improve GLM docs and examples

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
* Minor changes to docs and formulation
* Tweaks to the plot of the matrices to be more readable

### CODE REVIEW

File: examples/04_glm_first_level_models/plot_first_level_model_details.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
-"""Studying first-level-model details in a trials-and-error fashion
-===================================================================
+"""Understanding parameters of the first-level model
+====================================================
 
-In this tutorial, we study the parametrization of the first-level
-model used for fMRI data analysis and clarify their impact on the
+In this tutorial, we study how first-level models are parametrized
+for fMRI data analysis and clarify thee impact of these parameters on the
```
Comment (bthirion) [inline examples/04_glm_first_level_models/plot_first_level_model_details.py:5]:
```suggestion
for fMRI data analysis and clarify the impact of these parameters on the
```

**Comment by GaelVaroquaux** [comment]
I've included your fix, @bthirion 

This is ready for merge.

**Comment by bthirion** [comment]
Waiting for green CI then

**Comment by bthirion** [comment]
thx !

**Pull request merged**